cmake/std/PullRequestLinuxCuda10.1.105uvmOffTestingSettings.cmake

```diff
@@ -145,7 +145,7 @@ set (Anasazi_ENABLE_TESTS OFF CACHE BOOL "Turn off tests for non-UVM build")
 set (Domi_ENABLE_TESTS OFF CACHE BOOL "Turn off tests for non-UVM build")
 set (Kokkos_ENABLE_TESTS OFF CACHE BOOL "Turn off tests for non-UVM build")
 set (KokkosKernels_ENABLE_TESTS OFF CACHE BOOL "Turn off tests for non-UVM build")
-set (NOX_ENABLE_TESTS OFF CACHE BOOL "Turn off tests for non-UVM build")
+# set (NOX_ENABLE_TESTS OFF CACHE BOOL "Turn off tests for non-UVM build")
```

Comment:
Instead of commenting out, can we just remove this line?

Reply:
Fixed